src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequest.java

```diff
@@ -165,9 +167,29 @@ public void check(GHPullRequest ghpr, boolean isWebhook) {
         // Call update PR with the update PR info and no comment
         updatePR(ghpr, null /*GHIssueComment*/, isWebhook);
         checkSkipBuild();
+        checkLabels();
         tryBuild();
     }
 
+    private void checkLabels() {
+        Set<String> labelsToIgnore = helper.getLabelsIgnoreList();
+        if (labelsToIgnore != null && !labelsToIgnore.isEmpty()) {
+            try {
+                for (GHLabel label : pr.getLabels()) {
```

Comment:
`pr.getLabels()` is the call that ends up re-writing the pr object with an incorrect url. We'll need to fix this area of code or benefit from an upstream fix.